packages/db-mongodb/package.json

```diff
@@ -33,6 +33,7 @@
     "prepublishOnly": "pnpm clean && pnpm turbo build"
   },
   "dependencies": {
+    "@aws-sdk/client-sso-oidc": "^3.614.0",
```

Comment:
Why was this one added? Part of the resolution issue?

Reply:
Sorry, just mentioned the wrong package in the PR description. Changed it to `@aws-sdk/client-sso-oidc`